src/qml/QFieldLocalDataPickerScreen.qml

```diff
@@ -168,7 +169,10 @@ Page {
 
           width: parent ? parent.width : undefined
           height: line.height
-          color: "transparent"
+          color: Theme.controlBackgroundColor
+          border.color: Theme.controlBorderColor
+          border.width: 1
+          radius: 2
```

Comment:
You'll also need to adjust the other visual improvements (i.e. green hyperlink -> default text color, smaller italic text -> smaller non-italic text)